usb.cpp

```diff
@@ -173,25 +172,33 @@ namespace bsp
         }
     }
 
-    int usbCDCSendRaw(const char *dataPtr, size_t dataLen)
+    size_t usbCDCSendRaw(const char *dataPtr, size_t dataLen)
```

Comment:
In this file you can use `std::size_t` ;)

Reply:
You're right, fixed! :)